src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp

```diff
@@ -86,6 +103,9 @@ QImage LightPathsWidget::capture()
 void LightPathsWidget::set_transform(const Transformd& transform)
 {
     m_camera_matrix = transform.get_parent_to_local();
```

Comment:
Don't you need to update the camera matrix here ?

Reply:
I do, unless you mean something other than changing the GL view matrix :P

Reply:
m_gl_camera_matrix = transpose(m_camera_matrix) ?

Reply:
`m_gl_camera_matrix` is basically what `m_gl_view_matrix` is, which does get updated